vite.config.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable sort-keys */
+
+import { defineConfig } from 'vite';
+import { version } from './package.json';
+
+export default defineConfig({
+    root: 'src',
+    build: {
+        outDir: '../dist',
```

Comment:
`../` doesn't seem right here?

Reply:
The outDir is relative to the root, our sources are in the "src" folder and we want the dist folder outside of it. Just stupid JavaScript ecosystem things.

Reply:
See https://vitejs.dev/config/build-options.html#build-outdir